src/Pages/firebase.js

```diff
@@ -0,0 +1,64 @@
+import { db } from '../lib/firebase';
+import { collection, getDocs, addDoc, doc } from 'firebase/firestore';
+import { useEffect, useState } from 'react';
+
+export default function Firebase() {
+  // const [item, setItem] = useState('');
```

Comment:
Even though this is "throw-away" code, we may still want to remove commented out code.